mmdet/core/bbox/transforms.py

```diff
@@ -3,6 +3,22 @@
 import torch
 
 
+def remove_outside_bboxes(bboxes, img_h, img_w):
```

Comment:
This function does not really remove the outside bboxes, suggest using find_inside_bboxes

Reply:
`find_inside_bboxes` sounds like the `bboxes` are completely inside the image?

Reply:
The function allows a part of `bboxes` is inside the image, meanwhile the other parts are out of boundary.

Reply:
Do we have a better name for the function?